spread.yaml

```diff
@@ -263,6 +268,11 @@ suites:
                 prepare_classic
             fi
         restore: |
+            if [ "$REMOTE_STORE" = staging ]; then
+                . $TESTSLIB/store.sh
+                teardown_staging_store
+            fi
+
             $TESTSLIB/reset.sh
```

Comment:
can't we change this somehow to  "reset.sh --store"  and move the lines above inside reset.sh ? the asymmetry with prepare (which does the setup_staging_store) is a bit strange to me